### ISSUE TITLE
Support angular 2.3.0?

### ISSUE THREAD

**Issue by jefbarn** [scttcper/toastr-ng2#36]
Looks like thinks are broken with Angular 2.3.0...
```
TypeError: this._appRef.registerChangeDetector is not a function
    at DomPortalHost.attachComponentPortal (dom-portal-host.js:41)
    at DomPortalHost.BasePortalHost.attach (portal.js:128)
    at OverlayRef.attach (overlay-ref.js:12)
    at ToastrService._buildNotification (toastr-service.js:111)
    at ToastrService.error (toastr-service.js:21)
...
```
Need to use attachView now?
http://stackoverflow.com/questions/39857222/angular2-dynamic-component-injection-in-root

**Comment by scttcper** [comment]
I was waiting for angular-cli to support angular 2.3.0. They seem to be tracking the status of that internally. https://github.com/angular/angular-cli/issues/3467 I guess I could try the downgrade suggestion for testing... I'm on the fence on how to support this while doing the least amount of work

**Comment by jefbarn** [comment]
Cool, I didn't know angular-cli was having problems.  I can fork and try the attachView method against my app and send you a PR if it works.

**Comment by scttcper** [comment]
https://github.com/angular/material2/blob/master/src/lib/core/portal/dom-portal-host.ts#L46-L76 it does look like material 2 supports the attachView you mentioned.

**Comment by jefbarn** [comment]
Tried to fork, but can't get my build to run (may just be a 2.3.0 dependency problem, I can't tell).  Seems to be hung up on AOT compiler stuff.
```
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.getPrototypeOf (<anonymous>)
    at ReflectionCapabilities.annotations (http://localhost:8080/app/app.main.df6bdc.js:130531:50)
    at Reflector.annotations (http://localhost:8080/app/app.main.df6bdc.js:130723:44)
    at DirectiveResolver.isDirective (http://localhost:8080/app/app.main.df6bdc.js:60031:61)
    at CompileMetadataResolver._getTypeDescriptor
 ...
```

**Comment by scttcper** [comment]
released under `npm i toastr-ng2@next` until angular-cli is updated

**Comment by scttcper** [comment]
that was fast. angular-cli is updated now Released under 3.2.1

**Comment by jefbarn** [comment]
Thanks, works great!
